pkgs/development/python-modules/py-stringmatching/default.nix

```diff
@@ -1,30 +1,40 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
+
+  # build-system
+  cython,
   setuptools,
-  pytestCheckHook,
-  numpy,
-  pythonOlder,
+
+  # dependencies
+  numpy_1,
   six,
+
+  # tests
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "py-stringmatching";
   version = "0.4.6";
-  pyproject = true;
-
-  disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-XdHLHwT/sgHM+uQ4lxw9c+AcAdJjL6OVgfwtJkYLoBs=";
+  src = fetchFromGitHub {
+    owner = "anhaidgroup";
+    repo = "py_stringmatching";
+    tag = "v${version}";
+    hash = "sha256-gQiIIN0PeeM81ZHsognPFierf9ZXasq/JqxsYZmLAnU=";
   };
 
-  nativeBuildInputs = [ setuptools ];
+  pyproject = true;
+
+  build-system = [
+    setuptools
+    cython
+  ];
 
-  propagatedBuildInputs = [
-    numpy
+  dependencies = [
+    numpy_1
```

Comment:
This is not allowed: https://nixos.org/manual/nixpkgs/stable/#contributing
fixed in #455705

Reply:
Thank you.